frontend/src/components/BaseStoryRoof/BaseStory2.js

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect } from 'react';
```

Comment:
This component seems to be almost the same as `BaseStory1.js` - it would be great to make them one if possible